src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__title {
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 4px;
+  color: #060b35;
+}
+
+.card__code {
+  width: fit-content;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 84px 16px 16px;
+}
+
+.card__rating {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+
+  display: inline-block;
+}
+
+.stars .stars-star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The class `.stars-star` is modified by `:nth-child(-n + 4)` to change the image, which suggests that these are active stars, but there is no `star-active.svg` provided in the changed files. Make sure to include all necessary assets if they are meant to be part of the pull request. If `star-active.svg` is not meant to be different from `star.svg`, consider removing this rule to avoid confusion.